server/routes/schedules.js

```diff
@@ -0,0 +1,21 @@
+const express = require("express");
+const router = express.Router();
+const conn = require("../mariadb");
+const { StatusCodes } = require("http-status-codes");
+const {
+  getAllSchedules,
+  getScheduleById,
+  addSchedule,
+  updateSchedule,
+  deleteSchedule,
+} = require("../controller/schedulesController");
+
+router.use(express.json());
+
+router.get("/", getAllSchedules);
+router.get("/detail/:id", getScheduleById);
+router.post("/", addSchedule);
+router.put("/:id", updateSchedule);
+router.delete("/delete/:id", deleteSchedule);
```

Comment:
Restful하게 API를 설계한다고 했을 때 delete는 포함되지 않아도 괜찮을 것 같네요!
```suggestion
router.delete("/:id", deleteSchedule);
```